Source/Eto.Gtk/Forms/Controls/SplitterHandler.cs

```diff
@@ -11,37 +11,193 @@ public class SplitterHandler : GtkContainer<Gtk.Paned, Splitter, Splitter.ICallb
 		SplitterOrientation orientation;
 		SplitterFixedPanel fixedPanel;
 		int? position;
+		double relative = double.NaN;
+		int suppressSplitterMoved;
 
 		class EtoHPaned : Gtk.HPaned
 		{
+			public SplitterHandler Handler { get; set; }
+
 			#if GTK2
 			protected override void OnSizeRequested(ref Gtk.Requisition requisition)
 			{
```

Comment:
Overriding `OnSizeRequested` in each Gtk.Paned removes the big `if (orientation = ...) {} else {}`